src/NexusMods.App.UI/Controls/Trees/Common/FileTreeNodeIconType.cs

```diff
@@ -105,6 +106,22 @@ public static FileTreeNodeIconType GetIconType(this Extension extension)
         FileTreeNodeIconType.Video => "VideoOutline",
         _ => ThrowArgumentOutOfRangeException(iconType),
     };
+    
+    /// <summary>
+    /// Provides the direct <see cref="IconValues"/> for the given icon type.
+    /// </summary>
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    public static IconValue GetIconValue(this FileTreeNodeIconType iconType) => iconType switch
+    {
+        FileTreeNodeIconType.File => IconValues.File,
+        FileTreeNodeIconType.ClosedFolder => IconValues.Folder,
+        FileTreeNodeIconType.OpenFolder => IconValues.FolderOpen,
+        FileTreeNodeIconType.Image => IconValues.Image,
+        FileTreeNodeIconType.Text => IconValues.FileDocumentOutline,
+        FileTreeNodeIconType.Audio => IconValues.MusicNote,
+        FileTreeNodeIconType.Video => IconValues.Video,
+        _ => ThrowArgumentOutOfRangeExceptionIcon(iconType),
+    };
```

Comment:
The method should be moved into the new provider, the entire `Controls/Trees` directory will be removed once you're done moving the old code to our new system.

Reply:
The provider isn't the right place to put this as the Preview Changes view will also use this code.

I am for moving it, but I want to do it as part of the cleanup step that involves removing the old code for the new code.

The same way that I lifted out some of the functionality from old ViewModel into helpers  but didn't include them in this PR.